docs/csharp/whats-new/csharp-14.md

```diff
@@ -0,0 +1,100 @@
+
+C# 14 is supported on **.NET 10**. For more information, see [C# language versioning](../language-reference/configure-language-version.md).
+
+You can download the latest .NET 10 SDK from the [.NET downloads page](https://dotnet.microsoft.com/download). You can also download [Visual Studio 2022](https://visualstudio.microsoft.com/vs/), which includes the .NET 10 SDK.
+
+New features are added to the "What's new in C#" page when they're available in public preview releases. The [working set](https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md#working-set) section of the [roslyn feature status page](https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md) tracks when upcoming features are merged into the main branch.
+
+You can find any breaking changes introduced in C# 14 in our article on [breaking changes](~/_roslyn/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%2010.md).
+
+[!INCLUDE [released-version-feedback](./includes/released-feedback.md)]
+
+## The `field` keyword
+
+The token `field` enables you to write a property accessor body without declaring an explicit backing field. The token `field` is replaced with a compiler synthesized backing field.
+
+For example, previously, if you wanted to ensure that a `string` property couldn't be set to `null`, you had to declare a backing field and implement both accessors:
+
+```csharp
+private string _msg;
+public string Message
+{
+    get => _msg;
+    set
+    {
+        if (value is null) throw new NullArgumentException(nameof(value));
+        _msg = value;
+    }
+}
+```
+
+You can now simplify your code to:
+
+```csharp
+public string Message
+{
+    get;
+    set
+    {
+        field = (value is not null) ? value : throw new NullArgumentException(nameof(value));
+    }
+}
+```
+
+You can declare a body for one or both accessors for a field backed property.
+
+There's a potential breaking change or confusion reading code in types that also include a symbol named `field`. You can use `@field` or `this.field` to disambiguate between the `field` keyword and the identifier, or you can rename the current `field` symbol to provide more distinction.
+
+[!INCLUDE[field-preview](../includes/field-preview.md)]
+
+If you try this feature and have feedback, comment on the [feature issue](https://github.com/dotnet/csharplang/issues/140) in the `csharplang` repository.
+
+The [`field`](../language-reference/keywords/field.md) contextual keyword is in C# 13 as a preview feature. You can try it if you're using .NET 9 and C# 13 to provide [feedback](https://github.com/dotnet/csharplang/issues/140).
+
+## Implicit span conversions
+
+C# 14 introduces first-class support for <xref:System.Span`1?displayProperty=fullName> and <xref:System.ReadOnlySpan`1?displayProperty=fullName> in the language. This support involves new implicit conversions allowing more natural programming with these integral types.
```

Comment:
Let's remove integral to avoid confusion with integer types.

```suggestion
C# 14 introduces first-class support for <xref:System.Span`1?displayProperty=fullName> and <xref:System.ReadOnlySpan`1?displayProperty=fullName> in the language. This support involves new implicit conversions allowing more natural programming with these types.
```